src/sql/magic.py

```diff
@@ -640,6 +674,38 @@ def _persist_dataframe(
         )
 
 
+def get_query_type(command: str):
+    """
+    Returns the query type of the original sql command
+    """
+    query_type = (
+        sqlparse.parse(command)[0].get_type() if sqlparse.parse(command) else None
+    )
+    if query_type == "UNKNOWN":
+        return None
+    return query_type
+
+
+def get_dependency_in_CTE(sql_original, dependencies):
+    """
+    The following bit of code checks for dependencies within the parentheses
```

Comment:
this docstring is pretty long, we can remove things like "the following bit of code checks"

Reply:
I've removed the function entirely and provided a better, more generic warning instead

Reply:
[Fixed](https://github.com/ploomber/jupysql/pull/878/files#diff-527caf90dd08297f5253e826bd98fea0b6f4fbd44fe9f20472976f77868dac06R449-R451)